src/containers/ArticlePage/TopicArticlePage/components/TopicArticleTaxonomy.tsx

```diff
@@ -303,6 +301,10 @@ class TopicArticleTaxonomy extends Component<Props, State> {
 
     return (
       <Fragment>
+        <TaxonomyConnectionErrors
+          articleType={article.articleType ?? 'topic-article'}
+          taxonomy={article.taxonomy}
+        />
```

Comment:
Eg trur eg ville lagt dette under sjekk på taxonomy:admin for dette kjem til å forvirre vanlige brukere av ed.